core/eval.go

```diff
@@ -820,11 +821,21 @@ func evalSTACKREFPEEK(args []string) []byte {
 // containing the new value of the key along with the operation that was performed on it.
 // Contains only one argument, the key to be watched.
 func evalQWATCH(args []string, c *Client) []byte {
-	if len(args) != 1 {
+	if len(args) < 2 {
 		return Encode(errors.New("ERR invalid number of arguments for `WATCH` command"), false)
 	}
 
-	var key string = args[0]
+	// Join the args to form the query.
+	// TODO: In future we should support additional options for the QWATCH
+	// command, in that case we will need more sophisticated argument parsing logic.
+	var sql string = strings.Join(args, " ")
```

Comment:
Looks like we are expecting people to pass SQL Query as

```
QWATCH select score:
```

Because of this we are would need to join the query args, I propose can we do it as a single string like

```
QWATCH "select score;"
```

This way the SQL query will be a single argument, making the code readable. Even RESP has the support for long strings so this won't be a problem serializing and deserializing.

Reply:
Done.